modules/reference/pages/systems/system-details/sd-virtualization.adoc

```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
 image::system_details_traditional_virtualization_deployment.png[scaledwidth=80%]
 
 When the deploment scheduled it is listed as an action on the menu:Main Menu[Schedule > Pending Actions].
+
+== Storage [Salt]
+
+The [guimenu]``Storage`` tab presents a tree list of the virtual storage pools and volumes defined on the virtual host.
+The first level of the tree is the list of storage pools and all items contained in them are volumes.
+By default the pools need to be expanded to show the volumes.
+
+image::modules/reference/assets/images/system_details_salt_virtual_pools.png[scaledwidth=80%]
+
+For each pool the following information is provided:
+
+[guimenu]``Status``::
+This field indicates whether the pool is running or stopped.
+
+[guimenu]``Autostart``::
+This field indicates whether the pool is started automatically when the virtual host boots
+
+[guimenu]``Persistent``::
+This field indicates whether the pool will be kept or not after being stopped.
+
+[guimenu]``Location``::
+This field indicates the target path of the storage pool.
+Note that some pool types don't have any associated path.
+
+[guimenu]``Usage``::
+This field shows the disk usage of the pool or [guimenu]``Unknown`` if the pool is not running.
+
+For each volume the following information is provided:
+
+* The names of the virtual machines using the volume.
+This list may not be provided depending on the pool type.
+
+* The disk usage of the volume.
+
+=== Refreshing a pool
+
+[systemitem]``Libvirt`` doesn't update the pool usage and contents in real time.
+In order to update the usage or to see a volume that has been created outside of {productname}, the pool needs to be refreshed.
+Click the [guimenu]``Refresh`` button to schedule a refresh of the pool.
+
+=== Creating a pool
+
+To create a storage pool click the [guimenu]``Create Pool`` button.
+This opens a new page with a form to define the pool.
+It contains the following fields:
+
+[guimenu]``name``::
+The name of the pool to create.
+
+[guimenu]``type``::
+The type of the pool to create.
+The list of available types depends on the virtual host setup.
+// TODO Add a note on the supported types pointing to SUSE Documentation
+
+[guimenu]``Start during virtual host boot``::
+When this box is checked the storage pool will be automatically started during the virtual host boot.
+
+[guimenu]``Earliest``::
+The earliest time the pool creation action should be scheduled.
+
+[guimenu]``Add to``::
+Add the pool creation action to a new or already created action chain.
+
+All the remaining fields are depending on the selected field type.
+These fields are grouped in two panels:
+
+* the [guimenu]``Source`` panel contains the data about the device holding the pool.
+* the [guimenu]``Target`` panel contains the data about where to find the pool on the virtual host.
+
+==== Source fields
```

Comment:
```suggestion
==== Source Fields
```

I'm not sure this needs to be a heading.

Reply:
It's a field group in the dialog. I leave it up to you how to group those in the doc, if we want them grouped there at all

Reply:
I think leave it how it is for now, I'll need to go over this section once it's in to do the final edit.